orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -348,7 +348,7 @@ std::unordered_map<std::string, std::unordered_map<std::string, py::object>> Con
 void addObjectMethodsForTraining(py::module& m, ExecutionProviderRegistrationFn ep_registration_fn) {
   py::class_<std::vector<OrtValue>>(m, "OrtValueVector")
       .def(py::init<>())
-      .def("push_back", [](std::vector<OrtValue>* v, const OrtValue& ortvalue) {
+      .def("push_back", [](std::vector<OrtValue>* v, const OrtValue& ortvalue, const bool /*to_make_api_consistant*/) {
```

Comment:
Consistent with which API?

Reply:
Is this really needed? I don't think I saw the third argument being passed in in any call in this PR

Reply:
it is used here:
https://github.com/microsoft/onnxruntime/blob/e2ee7a734e7b4acd95ef6d27e753411afcdb8657/orttraining/orttraining/python/training/ortmodule/_training_manager.py#L51

but as discussed in other places, if we don't want the "to_ort_accepted_tensors" return different types, and move the branches to the caller, then we don't need this,

Reply:
removed.